lib/streamlit/commands/page_config.py

```diff
@@ -102,3 +106,38 @@ def set_page_config(
     if ctx is None:
         return
     ctx.enqueue(msg)
+
+
+def get_random_emoji():
+    import random
+
+    # Emojis recommended by https://share.streamlit.io/rensdimmendaal/emoji-recommender/main/app/streamlit.py
+    # for the term "streamlit". Watch out for zero-width joiners,
+    # as they won't parse correctly in the list() call!
+    RANDOM_EMOJIS = list(
+        "🔥™🎉🚀🌌💣✨🌙🎆🎇💥🤩🤙🌛🤘⬆💡🤪🥂⚡💨🌠🎊🍿😛🔮🤟🌃🍃🍾💫🥵▪🌴🎈🎬🌀🎄😝☔⛽🍂💃😎🍸🎨🥳☀😍🅱🌞😻🌟😜💦💅🦄😋😉👻🍁🤤😩👯🌻‼🌈👌🎃💛😚🔫🙌👽🍬🌅☁🍷👭☕🌚💁👅🥰🍜😌🎥🕺❕🧡☄💕🍻✅🌸🚬🤓🍹®☺💪😙☘🙅🤠✊🤗🍵🤞😂💯😏📻🎂💗💜🌊❣🌝😘💆🤑🌿🦋😈⛄🚿😊🌹🥴😽💋😭🖤🙆👐⚪💟☃🙈🍭💻🥀🚗🤧🍝💎💓🤝💄💖🔞⁉😫⏰🕊🎧☠♥🌳🏾🙉⭐💊🍳🌎🙊💸❤🔪😆🌾✈📚💀🏠✌🏃🌵🚨💂🤫🤭😗😄🍒👏🙃🖖💞😤😅🎅🍄🆓👉💩🔊🤷🤯⌚👸😇🚮💏👳🏽💘💿💉👠🎼🎶🎤👗❄🔐🎵🤒🍰👓🏄🌲🎮🙂📈🚙📍😵🗣❗🌺🙄👄🚘🥺🌍🏡♦💍🌱👑👙☑👾🍩🥶📣😓🏼🤣☯👵🍫➡🎀😃✋😬🍞🙇😹🙏👼🐝⚫🎁🍪🔨🌼👆👀😳🌏📖👃🎸👧😢💇🔒💙😞⛅🏻🍴😼😣🗿🍗♠🦁✔🤖😪☮🐢👎🐎💤😀🍺😁😴📺☹😲😖👍🎭💚🍆🍋🔵😥🏁🔴🔔😰🧐🤢👰☎🏆🤡🐠😒📲🙋📌🐬✍🔑📱💰🤕🐱🤦💧🎓🍕👟😷🐣👫🍑😸🍦👁🆗😦🎯📢😧🚶🦅🐧💢🏀😕🚫💑🐟🌽🏊🍟😱😔💝🙀🤥🤨😮💲🐍🍥😐🐸☝♣👊😯⚓❌🐯😶🏿🏈📰🌧👿🐳💷🐺📞🆒🍀🤐🤮😟🚲🍔👹🙍🌷🙎🐥💵🔝📸⚠❓🎩✂🍼😑⬇⚾🍎💔🐔⚽💭🏌🐷🍍✖🍇📝🍊🐙😨👋🤔🥊🙁🗽🐑🐘🐰😠💐🐴♀🐦🍓✏👂🏴👇🆘😡🏉👩💌😺✝🐼🤬😿😾🐒🐶👺🖕👬🍉🐻🐾⬅⏬▶👮🍌♂🔸👶🐮👪⛳🐐🎾🐕👴🐨🐊🔹©🎣👦👣👨👈💬⭕📹📷"
```

Comment:
We should probably remove 'negative' ones. This doesn't seem very great
![image](https://user-images.githubusercontent.com/24946400/103851595-26ab7300-5078-11eb-85f9-2d87f1cc3c92.png)